scripts/protocgen.sh

```diff
@@ -43,7 +43,7 @@ buf protoc \
 --doc_out=./docs/core \
 --doc_opt=./docs/protodoc-markdown.tmpl,proto-docs.md \
 $(find "$(pwd)/proto" -maxdepth 5 -name '*.proto')
-go mod tidy
+# go mod tidy
```

Comment:
reminder to undo this change

Reply:
I personally like the fact that this runs w/o building. It's nice for refactoring and when being bandwidth sensitive. Maybe add this back in a make command?